addon/components/frost-modal-dialog.js

```diff
@@ -28,13 +28,36 @@ export default Component.extend({
   cancelAlias (confirmAlias) {
     return confirmAlias ? 'Cancel' : 'Close'
   },
+  didRender () {
+    Ember.$('.ember-remodal.window').addClass('frost-modal-dialog')
+  },
+
   actions: {
-    cancel () {
-      this.sendAction('dismiss')
-    },
     confirm () {
-      this.sendAction('confirm')
-      this.sendAction('dismiss')
+      this.get('onConfirmHandler')()
+    },
+    modalOpen () {
+      // this.set('$containerEl', Ember.$('.ember-remodal.window.remodal-is-opened .ps-container'))
```

Comment:
depending on UX feedback we may use the same scrolling setup+style as used in frost-modal-input